src/plugins/vis_type_timeseries/server/lib/vis_data/get_table_data.js

```diff
@@ -41,19 +40,27 @@ export async function getTableData(req, panel) {
 
   try {
     const body = buildRequestBody(req, panel, esQueryConfig, indexPatternObject, capabilities);
-    const [resp] = await searchRequest.search([
+    const [resp] = await searchStrategy.search(req.framework.core, req, [
       {
         body,
         index: panelIndexPattern,
       },
     ]);
-    const buckets = get(resp, 'aggregations.pivot.buckets', []);
+
+    const buckets = get(
+      resp.rawResponse ? resp.rawResponse : resp,
+      'aggregations.pivot.buckets',
+      []
+    );
+
+    console.log(buckets);
 
     return {
       ...meta,
       series: buckets.map(processBucket(panel)),
     };
   } catch (err) {
+    console.log(err);
```

Comment:
Seems you left these consoles unintentionally?

Reply:
removed